app/views/two-factor-auth/configure.njk

```diff
@@ -71,12 +71,9 @@
         name: "code",
         classes: "govuk-input--width-10",
         type: "text",
-        attributes: {
-          "autocomplete": "off",
-          "autofocus": true,
-          "inputmode": "numeric",
-          "pattern": "[0-9]*"
-        }
+        autocomplete: "off",
+        inputmode: "numeric",
+        pattern: "[0-9]*"
```

Comment:
Do these no longer need to be inside the `attributes` block?

Reply:
Correct, all three are now top-level options of the Nunjucks macro — this can be seen visiting https://design-system.service.gov.uk/components/text-input/ then selecting ‘Nujucks’ under the first example, followed by ‘Nunjucks macro options’.